py/strato/monitors/exceptionfinder.py

```diff
@@ -0,0 +1,213 @@
+import json
+from strato.common.log import config as logconfig
+import os
+import datetime
+import re
+import calendar
+import yaml
+import logging
+import time
+from collections import defaultdict
+
+FILEBEAT_CONFIG_FILE_PATH = 'tools/monitoring/filebeat.yml'
```

Comment:
i'd rather move it to monitors repo, as it doesn't have any direct connection to the strato-log reader